internal/argocd/cluster/manager.go

```diff
@@ -133,7 +133,7 @@ func (m *Manager) Start() error {
 	// To avoid this we need to re-save same info before cache is expired.
 	go func() {
 		for {
-			go m.refreshClusterConnectionInfo()
+			go m.refreshClusterInfo()
```

Comment:
Nit, and optional, up to you: Can likely remove `go` here, since it's already running in a `go` func. 

(Also optional: we could also convert the time.Sleep to a ticker, but since this change was already in the code, it's optional for this PR)